_cucumber/cucumber.rake

```diff
@@ -27,7 +27,17 @@ task :features do |t, args|
     profile = 'parallel'
   end
 
-  exit_status = runner.run(profile)
+  if ENV['CUCUMBER_TAGS'].to_s.empty?
+    tags = nil
+  else
+    tags = ENV['CUCUMBER_TAGS']
+    known_good_cucumber_tags = %w(@wip @smoke @cdk_beta @ignore)
+    unless known_good_cucumber_tags.include?(tags)
+      raise("#{tags} is not a known tag. If required it can be added to the known good cucumber tags in #{File.dirname(__FILE__)}/cucumber.rake!")
+    end
+  end
+
+  exit_status = runner.run(profile, tags)
   p "exit status was #{exit_status}"
```

Comment:
OK, so with multi tags the problem is this

irb(main):001:0> known_good_cucumber_tags = %w(@wip @smoke @cdk_beta @ignore)
=> ["@wip", "@smoke", "@cdk_beta", "@ignore"]
irb(main):002:0> known_good_cucumber_tags.include?("@wip @smoke")
=> false